Modules/Devices/src/Devices.Application/Infrastructure/PushNotifications/NotificationIdAttribute.cs

```diff
@@ -2,10 +2,15 @@ namespace Backbone.Modules.Devices.Application.Infrastructure.PushNotifications;
 
 public class NotificationIdAttribute : Attribute
 {
-    public NotificationIdAttribute(int value)
+    public NotificationIdAttribute(string value)
     {
+#if DEBUG
+        if (value.Length > 64)
+            throw new Exception("The value of the NotificationIdAttribute must be less than or equal to 64 characters.");
+#endif
```

Comment:
what's that?

Reply:
That's because APNs allows not more than 64 characters for the x-collapse-id header (see https://developer.apple.com/library/archive/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/CommunicatingwithAPNs.html for reference)

Reply:
But Why is this enabled only for debugging?

Reply:
I had a stupid idea, which doesn't make any sense. :) Removed it.